### ISSUE TITLE
underscores in git email address breaks ticgit.

### ISSUE THREAD

**Issue by dokipen** [schacon/ticgit#6]
The problem is on line 46 of lib/ticgit/ticket.rb

```
      # matching
      data = fname.split('_')
      if data[0] == 'ASSIGNED'
        t.assigned = data[1]
```

**Comment by dokipen** [comment]
Side effects that I've seen are:
- Unable to modify a ticket.
- Assign field only displays address up to the underscore.

**Comment by jeffWelling** [comment]
Commented out incomplete spec
- Closed by b20d53c66793e181ac485502595a445e87d11207
